src/Http/Http/src/DefaultHttpContext.cs

```diff
@@ -92,16 +115,43 @@ public void Uninitialize()
         private IHttpRequestIdentifierFeature RequestIdentifierFeature =>
             _features.Fetch(ref _features.Cache.RequestIdentifier, _newHttpRequestIdentifierFeature);
 
+        /// <summary>
```

Comment:
You can actually tell the docs system to just copy the docs from the base-class by using a single `<inheritdoc />` element:

```csharp
/// <inheritdoc />
public override IFeatureCollection Features => _features.Collection ?? ContextDisposed();
```

Reply:
You can do this for all of the `override` methods, since HttpContext has pretty good docs already: https://github.com/aspnet/AspNetCore/blob/master/src/Http/Http.Abstractions/src/HttpContext.cs